packages/chainspec/src/BaseParser.ts

```diff
@@ -29,20 +29,20 @@ const isSi = <T extends TypeDef>(
 ): type is T & { info: TypeDefInfo.Si; type: `Lookup${number}` } =>
   type.info === TypeDefInfo.Si && /^Lookup\d+$/.test(type.type);
 
-export type PrimitiveType = { type: 'primitive'; name: string };
+export type PrimitiveType = { type: 'primitive'; value: string };
```

Comment:
changed to `value` to match the other types and make it clearer that `value` is breaking